src/context.cpp

```diff
@@ -403,6 +404,25 @@ namespace librealsense
         }
     };
 
+    //A facade for a realdds::dds_option exposing librealsense interface
+    class rs2_dds_option : public option_base
+    {
+        std::shared_ptr< realdds::dds_option > _dds_opt;
+        std::string _stream_name;
+
+    public:
+        rs2_dds_option( const std::shared_ptr< realdds::dds_option > & dds_opt )
+            : option_base( { dds_opt->get_range().min, dds_opt->get_range().max,
+                             dds_opt->get_range().step, dds_opt->get_range().default_value } )
+            , _dds_opt( dds_opt )
+        {
+        }
+
+        void set( float value ) override {}; //TODO - implement
+        float query() const override { return 0.0f; }; //TODO - implement
```

Comment:
Why TODO?
Is this meant to be overwritten? So why not leave =0?
If not, why not forwarding to the dds option?

Reply:
It is pure virtual, will be overridden in next PR when setting/getting options (already implemented and waiting after this PR)